src/screens/Signup.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  StyleSheet,
+  Text,
+  View,
+  Button,
+  TextInput,
+  TouchableOpacity,
+} from 'react-native';
+import { useState } from 'react';
+
+export default function SignUpForm() {
+  const [email, setEmail] = useState('');
```

Comment:
Good work on properly using these hooks! Soon we'll be adding the functionality to it from [Annie's PR](https://github.com/calblueprint/vouchers4veggies/pull/11) along with making sure that the buttons route to the proper screens